AtlasExplorer can't find bzlib.h while compiling (Bzip2 build from source, installed in custom directory)

[ 87%] Building CXX object plugins/AtlasExplorer/CMakeFiles/braynsAtlasExplorer.dir/io/nrrdloader/data/decompressors/BZip2Decompressor.cpp.o

Brayns/plugins/AtlasExplorer/io/nrrdloader/data/decompressors/BZip2Decompressor.cpp:25:10: fatal error: bzlib.h: No such file or directory
   25 | #include <bzlib.h>
      |          ^~~~~~~~~
compilation terminated.

I built Bzip2 from source ( gitlab.com/bzip2/bzip2 ), and installed in a custom directory

--- a/plugins/AtlasExplorer/CMakeLists.txt
+++ b/plugins/AtlasExplorer/CMakeLists.txt
@@ -12,7 +12,8 @@ add_library(braynsAtlasExplorer SHARED ${BRAYNSATLASEXPLORER_SOURCES} ${BRAYNSAT
 target_include_directories(braynsAtlasExplorer PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>"
     "$<INSTALL_INTERFACE:include>")
-target_link_libraries(braynsAtlasExplorer PRIVATE braynsPluginAPI braynsNetwork ${ZLIB_LIBRARIES})
+target_link_libraries(braynsAtlasExplorer PRIVATE braynsPluginAPI braynsNetwork ${ZLIB_LIBRARIES} ${BZIP2_LIBRARIES})
+target_include_directories(braynsAtlasExplorer PRIVATE ${ZLIB_INCLUDE_DIRS} ${BZIP2_INCLUDE_DIRS})
 
 target_compile_options(braynsAtlasExplorer PRIVATE ${BRAYNS_COMPILE_OPTIONS})
 

